lib/cdo/graphics/certificate_image.rb

```diff
@@ -50,7 +49,6 @@ def apply_text(image, text, pointsize, font, color, x_offset, y_offset)
     Honeybadger.notify(
       exception,
       context: {
-        image_path: image_path,
```

Comment:
This is the line of code which was generating the Honeybader reports.

Reply:
I'm surprised this was causing an error without entering this code path?